### PR TITLE
feat: Add price in USD to auction results list item

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [CX-1707]

## Description

Adds the price in US$ to auction results if the price is available and shown in another currency.

[CX-1707]: https://artsyproduct.atlassian.net/browse/CX-1707
Screenshot 2021-08-26 at 15 08 28

### CODE REVIEW

File: src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx [hunk 1]
```diff
@@ -371,13 +384,28 @@ const renderPricing = (
...
+            <Flex alignItems="center" justifyContent="flex-end">
+              <Text variant="md" fontWeight="bold">
+                {salePrice}
+              </Text>
+              {!!showPriceUSD && (
+                <Text
+                  variant="xs"
+                  lineHeight={1}
+                  color="black60"
+                  style={{ paddingTop: "2px" }}
```
Comment (olerichter00) [inline src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:402]: Couldn't find a way a better way to have `salePrice` and `salePriceUSD` be aligned correctly without applying top padding. Any ideas?

Comment (damassi) [reply src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:402]: Only suggestion would be to remove the `lineHeight` prop above, as that might be throwing off the `alignItems=center` on the flex container; those values are baked into the variants and shouldn't be adjusted if possible. 

@dzucconi might have some suggestions!

**Review by damassi — APPROVED** [review]

File: src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx:402]: Setting `lineHeight={1}` on both should work fine.

Comment (olerichter00) [reply src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx]: Thanks for the help! Adding `alignItems="flex-end"` to the parent container did the trick 🎉

**Comment by olerichter00** [comment]
#squashongreen

**Pull request merged**